crates/consensus/beacon/src/engine/message.rs

```diff
@@ -146,6 +147,8 @@ pub enum BeaconEngineMessage {
     NewPayload {
         /// The execution payload received by Engine API.
         payload: ExecutionPayload,
+        /// The parent beacon block root, if any.
+        parent_beacon_block_root: Option<H256>,
```

Comment:
I thought that we could introduce a new wrapper type here, but I actually prefer having a `Some|None` argument in all the on_new_payload functions so it is evident when we're using it